content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -64,11 +64,10 @@ When a Container lifecycle management hook is called,
 the Kubernetes management system executes the handler according to the hook action,
 `httpGet` , `tcpSocket` and `sleep` are executed by the kubelet process, and `exec` is executed in the container.
 
-Hook handler calls are synchronous within the context of the Pod containing the Container.
-This means that for a `PostStart` hook,
-the Container ENTRYPOINT and hook fire asynchronously.
-However, if the hook takes too long to run or hangs,
-the Container cannot reach a `running` state.
+Hook handler calls like `PostStart`, are initiated when a container is created,
```

Comment:
```suggestion
Hook handler calls such as `PostStart`, are initiated when a container is created,
```

Reply:
It makes sense because I am introducing only one example `PodStart`, and not comparing it with others. Thanks!